h3-storage-and-processing/scratch.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
A little scratch notebook for playing with minio package since return objects in the package are not typed (intelisence says `Any`), so at first, I wanted to check it myself. Saved this notebook only for history.